azkaban-common/src/main/java/azkaban/project/DirectoryFlowLoader.java

```diff
@@ -151,6 +165,58 @@ public void loadProjectFlow(final Project project, final File baseDirectory) {
 
   }
 
+  private ArrayList<AzkabanFlow> convertYamlFileToAzkabanFlows(File projectDir) {
+    ArrayList<AzkabanFlow> azkabanFlows = new ArrayList<>();
+    // Load all yaml files to AzkabanFlows in current project directory,
+    // Do not support loading from subdirectories for now.
+    final File[] yamlFiles = projectDir.listFiles(new SuffixFilter(YML_SUFFIX));
+    Arrays.sort(yamlFiles);
```

Comment:
why do we need sort here?

Reply:
Probably not needed. I'm just following what the current code is doing for .properties files.